picframe/model.py

```diff
@@ -26,6 +26,7 @@
         'show_text': "name location",
         'text_justify': 'L',
         'text_bkg_hgt': 0.25,
+        'text_opacity': 255,
```

Comment:
In the .yaml file it is 1.0 ?

Reply:
Yes - the stored values match the format of the function parameter, the fractional value 0.0-1.0 is converted into this format 0-255 as the config is parsed.

We could have it as a float, it would just mean that the conversion would be performed each time the value was used rather than once

Reply:
Maybe I'm blind. If `text_opacity`is taken from the .yaml file you do `255 * text_opacity = 255 * 1.0`.
But in model.py the default is `255`. This results in `255 * text_opacity = 255 *255`
????

Reply:
I think I see what you mean - the defaults here pre-populate a config that is then overridden by the yaml values

I was looking at the number of other calculations in the app, perhaps the clarity of storing the value in the float and converting it to the right format at the point of use removes a point of confusion